packages/indexer-agent/src/agent.ts

```diff
@@ -29,6 +29,7 @@ import pMap from 'p-map'
 import pFilter from 'p-filter'
 import { ReceiptCollector } from './query-fees'
 
+// are these lists ordered? why a list?
```

Comment:
Are you proposing a set instead?

Reply:
I don't think it matters, just anxious if there was a particular order that I wasn't aware of